oscar/Courses-AT05/src/main/java/courses/sampleapp/ui/HomePage.java

```diff
@@ -0,0 +1,82 @@
+package courses.sampleapp.ui;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by Administrator on 11/15/2017.
+ */
+public class HomePage extends BasePageObject {
+
+  @FindBy(css = "p.message-text")
+  WebElement welcomeMessage;
+
+  @FindBy(css = "span.user-name")
+  WebElement nameProfile;
+
+  @FindBy(linkText = "My Dashboard")
+  WebElement title;
+
+  @FindBy(linkText = "Sign Out")
```

Comment:
this locator will not be valid before the internationalization of the application, I would ask you to change it

Reply:
ok...I will use xpath to find that elements